data/json/recipes/armor/feet.json

```
@@ -579,5 +579,18 @@
     "book_learn": [ [ "manual_tailor", 2 ] ],
     "using": [ [ "sewing_standard", 4 ] ],
     "components": [ [ [ "leather", 4 ] ] ]
+  },
+  {
+    "result": "sandal_armor",
+    "type": "recipe",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_FEET",
+    "skill_used": "tailor",
+    "difficulty": 3,
+    "time": "30 m",
+    "autolearn": true,
+    "book_learn": [ [ "textbook_armwest", 2 ] ],
+    "using": [ [ "sewing_standard", 8 ] ],
+    "components": [ [ [ "leather", 4 ] ] ]
```

```suggestion
    "components": [ [ [ "leather", 6 ] ] ]
```
Could afford to use a bit more material than normal leather sandals since these are basically up-armored caligae.